src/EditorFeatures/CSharpTest/UseIndexOrRangeOperator/UseRangeOperatorTests.cs

```diff
@@ -221,6 +219,94 @@ void Goo(string s, string t)
     {
         var v = t.Substring(s[1..^1][0], t.Length - s[1..^1][0]);
     }
+}", parseOptions: s_parseOptions);
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsUseRangeOperator)]
+        public async Task TestWithTypeWithActualSliceMethod1()
```

Comment:
💡 Add a test for the case where there is also a `Slice(int)` overload, and the value is `0`

Reply:
➡️ I misread the spec. This is not necessary.

Reply:
Ok.  Note that this is not found by the pattern due to:

https://github.com/dotnet/csharplang/blob/master/proposals/csharp-8.0/ranges.md#implicit-range-support

> The type has an accessible member named Slice which has two parameters of type int.

Will add tests though.